### DISCUSSION TITLE
Itwin Datasmith Exporting Error

### DISCUSSION THREAD

**Discussion by KBStrt** [iTwin/itwinjs-core#4383] [Q&A]
Hi all, 

I'm having issues while exporting any model to datasmith. I really don't know what's happening, it closes without any warning.

**Comment by calebmshafer** [comment]
@danieliborra @MichaelBelousov @mgooding

**Comment by mgooding** [comment]
A few questions:
- Have you tried uninstalling and reinstalling?
- If you create a copy of one of the sample iModels using developer.bentley.com and "My iModels", do you still see this problem when exporting that?

**Comment by MichaelBelousov** [comment]
@KBStrt 
Could you confirm the version you are running by hitting the help button in the top right and pressing the "Open debug log" option? The log should have a version at the top (if there is no version at the top of the log, you are on version 0.7.1 which had a regression and didn't log the version number).

Since the iTwin Exporter for Datasmith updates automatically, you are probably on the latest, 0.7.2, 0.7.1 had a regression while exporting models filtered by saved views, which 0.7.2 fixed. Are you sure you're on the latest version? Reinstalling (as mgooding said) would ensure that.

If you still have issues, could you open the debug log as instructed above, run the export until it closes/crashes, then send the contents of the debug log to me at mike.belousov@bentley.com?
